### PR TITLE
Add new data class strings from GateHub breach

### LINKED ISSUE #1385: Missing data classes for latest GateHub breach
Re: https://monitor.firefox.com/breach-details/GateHub

```sh
$ npx pdehaan/blurts-server-data-classes

Missing "cryptocurrency-wallet-hashes" data class from GateHub breach
Missing "mnemonic-phrases" data class from GateHub breach
```

Looks like we're missing some new data classes in [/locales/en/data-classes.ftl](https://github.com/mozilla/blurts-server/blob/master/locales/en/data-classes.ftl).

### PR DESCRIPTION
**Pull request by pdehaan**
Ref: https://monitor.firefox.com/breach-details/GateHub

Fixes #1385

### CODE REVIEW

**Comment by pdehaan** [comment]
@flodolo I found these via <kbd>npx pdehaan/blurts-server-data-classes</kbd>, so I think they're the only missing data class strings from a recent breach:

```
Missing "cryptocurrency-wallet-hashes" data class from GateHub breach
Missing "mnemonic-phrases" data class from GateHub breach
```

File: locales/en/data-classes.ftl [hunk 1]
```diff
@@ -64,6 +65,7 @@ ip-addresses = IP addresses
 job-titles = Job titles
 mac-addresses = MAC addresses
 marital-statuses = Marital statuses
+mnemonic-phrases = Mnemonic phrases
```
Comment (flodolo) [inline locales/en/data-classes.ftl]: This is impossible to understand for most people. Let's add a comment

```
# Mnemonic phrases are a group of words used to access the content of cryptocurrency wallets.
```

Comment (pdehaan) [reply locales/en/data-classes.ftl]: Awesome, thanks. I can do that tomorrow when I’m at my laptop, but are we ok to merge after that, or would you like me to do the other “en-*” locales while I’m at it?

**Comment by flodolo** [comment]
Just en-US works. Do you want we to add the comment and merge?

**Comment by groovecoder** [comment]
Looks like the comment was added. Merging.

**Pull request merged**